### DISCUSSION TITLE
Usable in Relation Manager

### DISCUSSION THREAD

**Discussion by e2d2-dev** [ysfkaya/filament-phone-input#17] [Ideas]
Hi, would be nice, if it would work in relationmanagers and modals too.

**Comment by ysfkaya** [comment]
It should work in relation manager and modals too. Because they both use Filament's form builder.

I haven't checked at the moment, but if you can share a sample schema of the problem I can take a look at it when available.

**Reply by e2d2-dev** [reply]
ok, i tested it again, works now. Sorry for bothering you, keep up the good work.
